src/Agent.Listener/CommandSettings.cs

```diff
@@ -62,13 +63,15 @@ public sealed class CommandSettings
             Constants.Agent.CommandLine.Args.WindowsLogonAccount,
             Constants.Agent.CommandLine.Args.WindowsLogonPassword,
             Constants.Agent.CommandLine.Args.Work,
-            Constants.Agent.CommandLine.Args.Yaml
+            Constants.Agent.CommandLine.Args.Yaml,
```

Comment:
.yml is the more common file extension name, seems natural for the command line arg to follow